### PR TITLE
Initial rubangarmon lily58

### PR DESCRIPTION
**Pull request by rubangarmon**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: readme.md
Comment (waffle87) [inline readme.md:1]: Changes to this file need to be reverted.

File: keyboards/lily58/keymaps/rubangarmon/config.h
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/config.h:1]: This file should be removed, as its contents are all the default behaviours (`MASTER_LEFT`/`TAPPING_TERM`), don't exist (`TAPPING_FORCE_HOLD`), or aren't used (every RGB related, as RGB is not enabled.

File: keyboards/lily58/keymaps/rubangarmon/keymap.c
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/keymap.c:1]: This file is missing a license header.

File: keyboards/lily58/keymaps/rubangarmon/keymap.c [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_BSPC,  QWERTY ,   KC_N,    KC_M,  KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+                            KC_LCTL, KC_LALT,  LOWER , KC_SPC ,  KC_ENT ,  RAISE, KC_RALT,  KC_LGUI
+    )    
+  */
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
```
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/keymap.c:192]: The [tri-layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used here instead.
```suggestion
```

File: keyboards/lily58/keymaps/rubangarmon/keymap.c [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  _RAISE,
+  _NUMER,
+  _FUNC,  
+  _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE
+};
+
```
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/keymap.c:15]: This has not been configured to do anything.
```suggestion
```

File: keyboards/lily58/keymaps/rubangarmon/keymap.c [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+#define ADJUS MO(_ADJUST)
+// #define NUMERL MO(_NUMER)
+#define LCTLESC LCTL_T(KC_ESC)
+// #define NUM_A LT(_NUMER, KC_A)
```
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/keymap.c:21]: Unused.
```suggestion
#define LCTLESC LCTL_T(KC_ESC)
```

File: keyboards/lily58/keymaps/rubangarmon/rules.mk [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+OLED_ENABLE= yes     # OLED display
+
+# If you want to change the display of OLED, you need to change here
```
Comment (waffle87) [inline keyboards/lily58/keymaps/rubangarmon/rules.mk:13]: Default behaviour.
```suggestion
```

File: keyboards/planck/keymaps/rubangarmon/readme.md
Comment (waffle87) [inline keyboards/planck/keymaps/rubangarmon/readme.md:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/lily58/keymaps/rubangarmon/config.h [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
```
Comment (drashna) [inline keyboards/lily58/keymaps/rubangarmon/config.h:25]: Default behavior, doesn't need to be specified.

File: keyboards/lily58/keymaps/rubangarmon/config.h [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 200
```
Comment (drashna) [inline keyboards/lily58/keymaps/rubangarmon/config.h:30]: tapping force hold is deprecated in favor of "quick tapping term".  And 200 is the default tapping term, so it doesn't need to be specified. 
```suggestion
```

File: keyboards/lily58/keymaps/rubangarmon/config.h [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+#undef RGBLED_NUM
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/lily58/keymaps/rubangarmon/config.h:33]: Deprecated.
```suggestion
```
Or
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/lily58/keymaps/rubangarmon/config.h [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+
+// Underglow
+/*
+#undef RGBLED_NUM
+#define RGBLED_NUM 14    // Number of LEDs
+#define RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_SLEEP
+*/
```
Comment (drashna) [inline keyboards/lily58/keymaps/rubangarmon/config.h:46]:
```suggestion
```

File: keyboards/lily58/keymaps/rubangarmon/config.h [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 200
+
+#undef RGBLED_NUM
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 27
```
Comment (drashna) [inline keyboards/lily58/keymaps/rubangarmon/config.h:34]: If it's 27 on one side, then this isn't the correct configuration for it.

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**